ank/src/cli_commands/workload_table_row.rs

```diff
@@ -94,4 +118,29 @@ mod tests {
             expected_table
         );
     }
+
+    #[test]
+    fn utest_additional_info_msg_without_new_lines() {
```

Comment:
To which requirement shall I link this utest? To `// [utest->swdd~cli-shall-present-workloads-as-table~1]` and leaving a comment in the requirement about the stripped info message?

Reply:
Ok done